custom/enikshay/two_b_release_1/management/commands/enikshay_2b_case_properties.py

```diff
@@ -0,0 +1,496 @@
+        changes = filter(None,
+            [self.migrate_person(person.person, person.occurrences, person.episodes)]
+            + [self.migrate_occurrence(occurrence, person.episodes) for occurrence in person.occurrences]
+            + [self.migrate_episode(episode, person.episodes) for episode in person.episodes]
+            + [self.migrate_test(test, person.person) for test in person.tests]
+            + [self.migrate_referral(referral, person.occurrences) for referral in person.referrals]
+            + [self.migrate_trail(trail, person.occurrences) for trail in person.trails]
+            + [self.open_secondary_owners(drtb_hiv, person.person, person.occurrences)
+               for drtb_hiv in person.drtb_hiv]
+            + [self.close_drtb_hiv(drtb_hiv) for drtb_hiv in person.drtb_hiv]
+        )
+        if self.commit:
+            self.factory.create_or_update_cases(changes)
+
+    @staticmethod
+    def get_open_occurrence(occurrences):
+        occurrences = [case for case in occurrences
+                       if not case.closed and case.type == CASE_TYPE_OCCURRENCE]
+        return occurrences[0] if occurrences else None
+
+    def migrate_person(self, person, occurrences, episodes):
+        self.total_persons += 1
+        occurrence = self.get_open_occurrence(occurrences)
+
+        if occurrence:
+            episodes = [case for case in episodes
+                        if not case.closed and case.type == CASE_TYPE_EPISODE
+                        and any([index.referenced_id == occurrence.case_id for index in case.indices])
+                        and case.dynamic_case_properties().get('episode_type') == "confirmed_tb"]
+            episode = episodes[0] if episodes else None
+        else:
+            episode = None
+
+        props = {
+            'enrolled_in_private': 'false',
+            'case_version': '20',
+            'area': person.get_case_property('phi_area'),
+            'language_code': 'hin',
+            'referred_outside_enikshay_date': person.get_case_property('date_referred_out'),
+            'referred_outside_enikshay_by_id': person.get_case_property('date_by_id'),
+        }
+        if episode:
+            props.update({
+                'current_episode_type': episode.get_case_property('episode_type'),
+                'alcohol_history': episode.get_case_property('alcohol_history'),
+                'alcohol_deaddiction': episode.get_case_property('alcohol_deaddiction'),
+                'tobacco_user': episode.get_case_property('tobacco_user'),
+                'occupation': episode.get_case_property('occupation'),
+                'nikshay_id': episode.get_case_property('nikshay_id'),
+                'phone_number_other': episode.get_case_property('phone_number_other'),
+            })
+
+        phone_number = person.get_case_property('phone_number')
+        if phone_number:
+            # TODO check if it already begins with 91 or has a certain length?
```

Comment:
`phone_number` also begins with a 0, so this will result in an invalid number. I would suggest parsing this with the `phonenumbers` library to be sure.